src/Checkbox/Checkbox.component.tsx

```diff
@@ -53,11 +59,26 @@ export type Props = {
   theme?: any;
 };
 
-const SDiv = styled.div`
-  display: inline-block;
-  + div[data-radiobutton] {
-    padding-left: 20px;
+const displayStyleRules: Function = (
+  displayStyle: 'inline' | 'stacked',
+  theme: any,
+) => {
+  console.log(theme);
+  if (displayStyle === 'inline') {
+    return {
+      display: 'inline-block',
+      'padding-right': theme.radioButton.inlineSpacing,
```

Comment:
This should be `theme.checkbox.inlineSpacing`.

Reply:
Fixed